src/pages/main/model/store.ts

```diff
@@ -12,3 +14,7 @@ location.subscribe((value) => {
   openProjects.set(value === '/projects');
   openAbout.set(value === '/about');
 });
+
+hotkeys.register(['control', 'z'], 'undo');
```

Comment:
enum from `handleHistoryHotkeys` should be used for aliases

Reply:
Obvious, thx